Middleware/likeCheck.js

```diff
@@ -0,0 +1,18 @@
+const likesdb=require('../Models/likes.model')
+
+const likeCheck=async (req,res,next)=>{
+    const {likeid}=req.params;
+    try{
+        if(await likesdb.findById(likeid)){
+            next()
+        }
+    }catch(error){
+        console.log(error);
+        return res.status(404).json({
+            ok:false,
+            message:"Data not found"
```

Comment:
In checks you can actually tell what is not found it will be easier to debug. here like not found

Reply:
Sure rishav I'll refactor